src/sort/sort.less

```diff
@@ -1,4 +1,5 @@
 .sort-pf {
+  display: inline-block;
```

Comment:
Without this setting, Sort control group will look like the following screenshot.
<img width="714" alt="screen shot 2017-03-07 at 11 00 47 am" src="https://cloud.githubusercontent.com/assets/5689075/23640084/46cfab54-0326-11e7-8780-aa6b3f312975.png">